Exam/CC1/src/exercice4/Reservation.java

```diff
@@ -0,0 +1,41 @@
+package exercice4;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Reservation {
+	private Date date;
+	private int num;
+	
+	public Reservation(int num, String date) throws ParseException {
+		SimpleDateFormat f = new SimpleDateFormat("dd/MM/yyyy"); 
+		this.date = f.parse(date);
+		this.num = num;
+	}
+	
+	public void setReservation(int num, String date) throws ParseException {
+		SimpleDateFormat f = new SimpleDateFormat("dd/MM/yyyy"); 
+		this.date = f.parse(date);
+		this.num = num;
```

Comment:
What does `num` stand for?
Shouldn't it be a **unique** identifier?
If so, it cannot be changed...